tests/conftest.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
+@pytest.fixture()
+def tmp_zarr(tmp_path, request):
```

Comment:
Perhaps use `tmp_zarrfile`, to clarify this yields a file-path?

Reply:
done :)